src/StudyApi/DependencyRegistrations/InfrastructureRegistration.cs

```diff
@@ -30,6 +31,12 @@ public static class InfrastructureRegistration
     {
         private static readonly string[] ProdEnvironmentNames = { "production", "prod", "live" };
 
+        private static bool IsProdEnvironment(string environmentName)
```

Comment:
Do we need to be this tolerant of the environment name? The configuration should be safe for any non-production environment and production settings are used only for `production`.

If we do need this...

Use extension method similar to how the framework implements things.
https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Extensions.Hosting.Abstractions/src/HostEnvironmentEnvExtensions.cs#L42

Also use the built in `IsEnvironment(string)` check...

`public static bool HasProductionLikeName(this IHostEnvironment env) => ProdEnvironmentNames.Any(x => env.IsEnvironment(x));`

Reply:
I was just following what is currently in play, I have updated to use the environment directly with the extension method